clang/include/clang/AST/Type.h

```diff
@@ -194,19 +358,27 @@ class Qualifiers {
     FastMask = (1 << FastWidth) - 1
   };
 
+  Qualifiers() : Mask(0), PtrAuth() {}
+
```

Comment:
```suggestion
```
Not needed; the default ctor will be automatically called and `Mask` has an inline initializer.